packages/api/src/index.ts

```diff
@@ -11,6 +11,12 @@ export enum CollectionId {
 export interface IFirestoreSurgery {
     name: string;
     description: string;
+    surgeryName?: string;
+    poBox?: string;
+    phone?: string;
+    email?: string;
+    city?: string;
+    address?: string;
```

Comment:
Én ezeket (és a poBoxot) is egybevenném "location" néven, és engedném, hogy csak várost, kerületet, vagy ha akarnak, teljes címet adjanak meg. Nem szeretném, ha túl bonyolult lenne kitölteni, és sok helyen (pl. egy kistelepülésen) nem is számít a pontos cím.

Reply:
Szerintem azért lenne fontos így kezelni, mert mindenki más formátumban fogja beírni a címet, és ha később valamilyen oknál fogva szeretnénk azt a központi adatbázist használni, hogy egyszerűen be tudják jelölni a saját rendelőjüket, akkor a már meglévő adatokat kézzel kell átírni vagy kibogarászni, ami isszonyat meló lehet. Persze ha az nem terv, akkor mehet egybe, csak optikailag fura lesz, hogy mindenki máshogy írja be a címet.

Reply:
Hmm hmm. Lehet, hogy ezen a ponton még egyben tartanám, aztán ha úgy döntünk, hogy majd használjuk az adatbázist, akkor mégis szétválasztjuk, kézzel átnézzük azt az 50-100 adatot és rendbetesszük az adatokat.

Amúgyis valahogy így dönteném el, hogy akarjuk-e az adatbázist csinálni: ha hirtelen eléri a 100 regisztrált háziorvost, és ebből mondjuk 50 megosztja az oltási listáját, akkor talán fontosabb lesz az adatbázis bekötése.